dask_glm/algorithms.py

```diff
@@ -319,7 +319,7 @@ def bfgs(X, y, max_iter=500, tol=1e-14, family=Logistic):
     return beta
 
 
-def proximal_grad(X, y, regularizer=L1, lamduh=0.1, family=Logistic,
+def proximal_grad(X, y, regularizer=L1(), lamduh=0.1, family=Logistic,
```

Comment:
I agree, should be change.  Also in addition, we could move the string logic to a function in `regularizers.py`